src/TurbulenceAveragingPostProcessing.C

```diff
@@ -705,6 +710,14 @@ TurbulenceAveragingPostProcessing::execute()
       compute_lambda_ci(avInfo->name_, s_all_nodes);
     }
     
+    if ( avInfo->computeMeanResolvedKe_ ) {
+      // need locally owned and active nodes
+      stk::mesh::Selector s_locally_owned_nodes
+        = metaData.locally_owned_part() & stk::mesh::selectUnion(avInfo->partVec_);
+      //& !(realm_.get_inactive_selector());
```

Comment:
I think this line shouldn't be commented out

Reply:
Yes, this was a WIP debug.. .I will fix and push.